modules/nested-elements/base/widget-nested-base.php

```diff
@@ -132,12 +132,10 @@ protected function content_template_single_repeater_item() {}
 
 	public function print_template() {
 		parent::print_template();
-		if ( $this->get_initial_config()['support_improved_repeaters'] ?? false ) {
-			?>
-			<script type="text/html" id="tmpl-elementor-<?php echo esc_attr( $this->get_name() ); ?>-content-single">
-				<?php $this->content_template_single_repeater_item(); ?>
-			</script>
-			<?php
-		}
+		?>
```

Comment:
Hey @asafdl , I know this is still a WIP, but I just want to leave some comments before I go on leave. We just need to remove checks for the Experiment, but the widget config need to stay as is. So this code change for example can be reverted.

In general, we need to support the "improved infra" and the "old infra" for BC with 3rd parties. Some 3rd Parties could have Nested widgets, but they haven't added support for the new infra yet, but their widgets should still function.

We will know they have added support for the new infra by checking the widget config such as `support_improved_repeaters`.

I hope this makes sense. Let me know if anything is unclear.

At the risk of being annoying, I will add comments to the rest of the changes, just to make fully clear my understanding of this task :)

Reply:
ok gotcha, thanks :)